### PR TITLE
Update JSON Object reference for commands.

### PR DESCRIPTION
**Pull request by slemeur**
Fixed JSON reference for `Command` object which was not up to date.

Signed-off-by: Stevan Le Meur <stevan.lemeur@gmail.com>

### CODE REVIEW

File: src/main/_docs/devops/devops-runtime-stacks-data-model.md [hunk 1]
```diff
@@ -400,13 +402,16 @@ JSON:
         "commandLine": "mvn clean install -f ${current.project.path}",
         "name": "build",
         "type": "mvn",
-        "attributes": {}
+        "attributes": {
+          goal:"Build"
```
Comment (azatsarynnyy) [inline src/main/_docs/devops/devops-runtime-stacks-data-model.md]: quotation marks are missed

File: src/main/_docs/devops/devops-runtime-stacks-data-model.md [hunk 2]
```diff
@@ -400,13 +402,16 @@ JSON:
...
+        }
       },
       {
         "commandLine": "mvn -f /projects/console-java-simple clean install",
         "name": "console-java-simple: build",
         "type": "mvn",
         "attributes": {
+          goal:"Build",
```
Comment (azatsarynnyy) [inline src/main/_docs/devops/devops-runtime-stacks-data-model.md]: quotation marks are missed

File: src/main/_docs/devops/devops-runtime-stacks-data-model.md [hunk 3]
```diff
@@ -415,6 +420,7 @@ JSON:
         "name": "console-java-simple: run",        
         "type": "mvn",
         "attributes": {
+          goal:"Run",
```
Comment (azatsarynnyy) [inline src/main/_docs/devops/devops-runtime-stacks-data-model.md]: quotation marks are missed

**Comment by slemeur** [comment]
Thanks @azatsarynnyy. Fixed.

**Pull request merged**